src/PresentationalComponents/CreateImageWizard/WizardStepUploadAzure.js

```diff
@@ -0,0 +1,83 @@
+import './WizardStepUploadAzure.scss';
+
+const WizardStepUploadAzure = (props) => {
+    return (
+        <>
+            <TextContent className="textcontent-azure">
+                <Title headingLevel="h2">Upload to Azure</Title>
+                <Text>
+            Image Builder will send an image to an authorized Azure account.
+                </Text>
+                <Title headingLevel="h3">OAuth permissions</Title>
+                <Text>
+            In order to use Image Builder to push images to Azure, Image Builder must
+            be configured as an authorized application, and given the role of &quot;Contributor&quot;.<br />
+            Image Builder must be authorized by an account owner.<br />
+                    <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow">
+                        <small>Learn more</small></a>
+                </Text>
+
+                <a href="https://login.microsoftonline.com/common/oauth2/authorize?client_id=b94bb246-b02c-4985-9c22-d44e66f657f4
+&redirect_uri=https%3A%2F%2Flogin.microsoftonline.com%2Fcommon%2Foauth2%2Fnativeclient">
+            Authorize Image Builder on Azure <ExternalLinkAltIcon />
+                </a>
+                <Title headingLevel="h4">Permission status</Title>
+                { (props.uploadAzure.azureAuthorized && <><ResourcesEmptyIcon /> Image Builder is not authorized</>) ||
+              <><CheckCircleIcon /> Image Builder is authorized</> }<br />
+                { (props.uploadAzure.azureRoleGranted &&
+                   <><ResourcesEmptyIcon /> Application &quot;RHImageBuilder&quot; does not have role of &quot;Contributor&quot;</>) ||
+                  <><CheckCircleIcon /> Application &quot;RHImageBuilder&quot; has role of &quot;Contributor&quot; </> }<br />
+            </TextContent>
+
+            <Title headingLevel="h3">Destination</Title>
+            <Text>
+            Your image will be uploaded to the resource group in the subscription you specify.
+            </Text>
+            <Form>
+                <FormGroup isRequired label="Tenant ID" fieldId="azure-destination-tenant-id"
+                    helperTextInvalid={ (props.errors['azure-destination-tenant-id'] && props.errors['azure-destination-tenant-id'].value) || '' }
+                    validated={ (props.errors['azure-destination-tenant-id'] && 'error') || 'default' }>
+                    <TextInput value={ props.uploadAzure.options.tenant_id || '' }
+                        type="text" aria-label="Azure destination tenant-id" id="azure-destination-tenant-id"
+                        data-testid="azure-destination-tenant-id" isRequired
+                        onChange={ value =>
+                            props.setUploadOptions('azure', Object.assign(props.uploadAzure.options, { tenant_id: value })) } />
+                </FormGroup>
+                <FormGroup isRequired label="Subscription" fieldId="azure-destination-subscription"
```

Comment:
```suggestion
                <FormGroup isRequired label="Subscription ID" fieldId="azure-destination-subscription"
```

Sadly, we need a subscription ID here.